src/config/file.rs

```diff
@@ -100,6 +100,20 @@ impl ConfigFile {
         }
     }
 
+    pub(crate) fn get_profile_names(config: &str) -> ConfigFileResult<Vec<String>> {
+        let mut profiles: Vec<String> = Vec::new();
+        if !config.is_empty() {
+            let config_file: ConfigFile = serde_yaml::from_str(&config)?;
+            profiles = config_file
```

Comment:
I think this will preserve the order the items appear in the config file. I don't know whether that's what people would expect or if a list of sorted values is. I don't have a strong opinion on it. I just wanted to make sure it was a deliberate decision.

Reply:
I'm not going to worry about order for now.